lua/go/dap.lua

```diff
@@ -68,14 +68,14 @@ local function keybind()
     --
   }
   if _GO_NVIM_CFG.dap_debug_gui then
-    keys["n|p"] = map_cr('<cmd>lua require("dapui").eval()'):with_noremap():with_silent()
-    keys["v|p"] = map_cr('<cmd>lua require("dapui").eval()'):with_noremap():with_silent()
-    keys["n|K"] = map_cr('<cmd>lua require("dapui").float_element()'):with_noremap():with_silent()
-    keys["n|B"] = map_cr('<cmd>lua require("dapui").float_element("breakpoints")'):with_noremap():with_silent()
-    keys["n|R"] = map_cr('<cmd>lua require("dapui").float_element("repl")'):with_noremap():with_silent()
-    keys["n|O"] = map_cr('<cmd>lua require("dapui").float_element("scopes")'):with_noremap():with_silent()
-    keys["n|a"] = map_cr('<cmd>lua require("dapui").float_element("stacks")'):with_noremap():with_silent()
-    keys["n|w"] = map_cr('<cmd>lua require("dapui").float_element("watches")'):with_noremap():with_silent()
+    keys["n|p"] = map_cr('<cmd>lua require("dapui").eval()<CR>'):with_noremap():with_silent()
```

Comment:
I do not think \<CR> is need here. map_cr will append \<CR> to the end of the command.
<img width="621" alt="image" src="https://user-images.githubusercontent.com/1681295/176863913-110c122f-f5b4-4500-8235-8e0879e5efab.png">
Did you see anything wrong?

Reply:
Ah I put it there cause for me the commands end up command mode, addring CR fixed it